functests/5_latency_testing/latency_testing.go

```diff
@@ -267,3 +270,22 @@ func getNegativeTests(toolToTest string) []latencyTest {
 	}
 	return testSet
 }
+
+func getToolArgsRegex(tst latencyTest) []string {
+	//this lists of args depend on the ones the latency tool runners adds to tool command in cnf-features-deploy.
+	switch tst.toolToTest {
+	case oslat:
```

Comment:
Shouldn't this be in quotes ("oslat")? same for all the cases below

Reply:
there is a constant for each tool name, just for simplicity and better maintenance.
https://github.com/openshift-kni/performance-addon-operators/blob/a6ae635a2d4d099c5412b24f42184634ce51f2ac/functests/5_latency_testing/latency_testing.go#L19